web/content/docs/benchmarks/CreepBGRa/CreepBRGa.md

```diff
@@ -0,0 +1,147 @@
+$R_u=8.314472 \mbox{J/(Kmol)}$ is the universal gas constant, and
+${ \sigma}_f$ is a stress scaling factor.
+
+Creep potential and creep strain
+================================
+
+Assume there is a creep potential $g^c$ and the creep induced strain
+rate is given by the flow rule
+$$\dot { \mathbf \epsilon}^c ({ \sigma})= \frac{2}{3}{\dfrac{\partial g^c}{\partial { \mathbf \sigma}}}
+$$ where ${ \sigma}$ is the equivalence stress
+defined by
+${ \sigma}={\sqrt{{\frac{3}{2}}}}{\left\Vert{\mathbf s}\right\Vert}$
+with ${\mathbf s}= { \mathbf \sigma}-\frac{1}{3}{ \sigma}{\mathbf I}$,
+the deviatoric stress. The creep strain rate is then expressed as
+$$\begin{gathered}
+\dot { \mathbf \epsilon}^c ({ \sigma})= \frac{2}{3} {\dfrac{\partial g^c}{\partial { \sigma}}}{\dfrac{\partial { \sigma}}{\partial { \mathbf \sigma}}}=\sqrt{{\frac{2}{3}}}{\dfrac{\partial g^c}{\partial { \sigma}}}\dfrac{{\mathbf s}}{{\left\Vert{\mathbf s}\right\Vert}}={\dfrac{\partial g^c}{\partial { \sigma}}}\dfrac{{\mathbf s}}{\sigma}
+\end{gathered}$$
+ The above creep strain rate expression
+must be valid for the problems with different dimension. Applying
+the creep rate equation to one dimensional leads to
+ $$\begin{gathered}
+\dot { \epsilon}^c = {\dfrac{\partial g^c}{\partial { \sigma}}}{\color{red} {\frac{2}{3}}}=Ae^{-Q/R_uT}\left(\dfrac{{ \sigma}}{{ \sigma}_f}\right)^m
+\end{gathered}$$ which says 
+
+$$\begin{gathered}
+ {\dfrac{\partial g^c}{\partial { \sigma}}}={\color{red}{\frac{3}{2}}}Ae^{-Q/R_u T}\left(\dfrac{{ \sigma}}{{ \sigma}_f}\right)^m
+\end{gathered}$$
+
+Therefore, the creep strain rate for multi-dimensional problems can be
+derived as $$\begin{gathered}
+ \dot { \mathbf \epsilon}^c ({ \sigma})={\color{red} {\sqrt{\frac{3}{2}}}}Ae^{-Q/R_uT}\left(\dfrac{{ \sigma}}{{ \sigma}_f}\right)^m\dfrac{{\mathbf s}}{{\left\Vert{\mathbf s}\right\Vert}}
+\end{gathered}$$
+
+Stress integration
+==================
+
+The above creep strain rate expression can be simplified as $$\begin{gathered}
+  \dot { \mathbf \epsilon}^c ({ \sigma}) = b {\left\Vert{\mathbf s}\right\Vert}^{m-1} {\mathbf s}
+\end{gathered}$$
+with
+$$b={\left(\dfrac{3}{2}\right)^{(m+1)/2}} \dfrac{Ae^{-Q/R_uT}}{{{ \sigma}_f}^m}$$
+
+The strain rate under creeping condition can be expressed as
+$$\begin{gathered}
+\dot { \mathbf \epsilon}= \dot { \mathbf \epsilon}^e + \dot { \mathbf \epsilon}^c
+\end{gathered}$$ Due to Hook’s law, the stress rate is
+given by 
+$$\begin{gathered}
+\dot { \mathbf \sigma}= \mathbf{C} \dot { \mathbf \epsilon}^e = \mathbf{C} (\dot { \mathbf \epsilon}- \dot { \mathbf \epsilon}^c) 
+\end{gathered}$$ 
+where
+$\mathbf{C}:= \lambda \mathcal{J} + 2G \mathbf I \otimes \mathbf I  $ 
+with $\mathcal{J}$ the forth order identity, $\mathbf I$ the second order identity,
+$\lambda$ the Lame constant,  $G$ the shear module, and $\otimes$  the tensor
+product notation.
+
+is a forth order tensor. Substituting equation and the expression of $C$
```

Comment:
fourth

Reply:
Corrected.

@chleh  Thanks for your comments.